src/platform/plugins/shared/unified_search/public/query_string_input/query_bar_top_row.tsx

```diff
@@ -729,7 +712,7 @@ export const QueryBarTopRow = React.memo(
       return (
         <EuiFlexItem
           grow={!shouldShowDatePickerAsBadge()}
-          style={{ minWidth: shouldShowDatePickerAsBadge() ? 'auto' : 320, maxWidth: '100%' }}
+          css={{ minWidth: shouldShowDatePickerAsBadge() ? 'auto' : 320, maxWidth: '100%' }}
```

Comment:
```suggestion
          style={{ minWidth: shouldShowDatePickerAsBadge() ? 'auto' : 320, maxWidth: '100%' }}
```

Reply:
Happened automatically because of this https://github.com/elastic/eui/issues/8708